platform/ethereum/bounce/collection.go

```diff
@@ -0,0 +1,89 @@
+package bounce
+
+import (
+	"strconv"
+
+	"github.com/trustwallet/blockatlas/pkg/blockatlas"
+	"github.com/trustwallet/golibs/coin"
+)
+
+var (
+	chainIdMap = map[uint]int{
```

Comment:
for bounce we only need support for BSC right?

Reply:
Yes, no need mapping?

Reply:
`ethereum/bounce/collection.go` since it's in bounce folder, it does not need to be aware of the ethereum, only bsc in this case

Reply:
done